capture-sdk/sdk/src/main/res/layout/gc_detection_error_layout.xml

```diff
@@ -0,0 +1,30 @@
+<androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/gc_medium"
+    android:background="@drawable/gc_detection_error_background"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <TextView
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/gc_detection_error_popup_dismiss_button"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toBottomOf="parent"
+        android:textColor="@color/gc_dark_02"
+        android:text="QR code scanning is currently unavailable. Please take a picture of your invoice instead."/>
```

Comment:
[Request] Please use a string resource and for German use: "QR-Code scannen ist derzeit nicht verfügbar. Bitte fotografieren Sie stattdessen Ihre Rechnung."

Reply:
Forgot to add string resources...

Reply:
This is interesting that sonarcloud gives "0 new issues", but I think it must scream about using hardcode strings